### ISSUE TITLE
Use Xresources colors in new rasi themes

### ISSUE THREAD

**Issue by binaryplease** [DaveDavenport/rofi#833]
Hello,

Question:
Is it possible to use colors defined in .Xresources with the new (rasi) theme format?
It would be nice to have to define the colors only once for my whole setup.

Thanks

**Comment by sardemff7** [comment]
It is not possible. And rofi will completely drop Xresources support in the future.
The rasi theme format is much more powerful and maintaining multiple (and partially incompatible) format generates a huge amount of work.

You can now use environment variables in the rasi theme format, if that is of any help to your use case.
If you really want to keep all your colours in one place, I would suggest a wrapper script for rofi that would check if your Xresources file is newer than your rofi theme file.
